testdata/src/stack-traces.plain.summary.log

```diff
@@ -0,0 +1,11 @@
+
+Failed scenarios:
+samples/stack-traces/stack-traces.feature:9 # A failing step
+
+1 Scenarios (1 failed)
+1 Steps (1 failed)
+0m 0.5s
+
+Error: BOOM
+samples/stack-traces/stack-traces.feature:10
```

Comment:
I don't understand why these are rendered separately from the non-passing scenarios.